plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/xtext/server/build/XStatefulIncrementalBuilder.java

```diff
@@ -81,7 +81,7 @@ protected void unloadResource(URI uri) {
 
 	/** Run the build. */
 	public XBuildResult launch() {
```

Comment:
For reference: we did not make extensive performance measurements because (1) the logic added here already exists in the other Xtext builder (so it's not something entirely new), (2) there exists a separate user story for performance of incremental building (see link in task description).
However, I did some simple tests in large real-world projects to make sure there isn't anything major.